test/dynamo/test_repros.py

```diff
@@ -945,6 +945,16 @@ def f(a, b):
         self.assertEqual(a_ref.grad, a_test.grad)
         self.assertEqual(b_ref.grad, b_test.grad)
 
+    def test_normal_strided(self):
+        @torch.compile(backend="eager")
+        def fn(op_args, op_kargs):
+            return torch.normal(*op_args, **op_kargs)
+
+        op_args = [2.0, 2.0, [2, 2, 2]]
+        op_kargs = {"dtype": torch.bfloat16, "layout": torch.strided}
```

Comment:
This would be a good OpInfo test. For all reference implementations of factory functions, check that strided layouts are accepted.

Reply:
Filled an issue https://github.com/pytorch/pytorch/issues/111222